angular.json

```diff
@@ -19,10 +19,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
             "aot": true,
-            "assets": [
```

Comment:
Did you make some changes in angular.json or is it just the formatting? In the second case, please undo this and exclude angular.json from the pull request.